app/lib/actions/katello/upstream_subscriptions/remove_entitlements.rb

```diff
@@ -3,6 +3,8 @@ module Katello
     module UpstreamSubscriptions
       class RemoveEntitlements < Actions::Base
         def plan(pool_ids = [])
+          ::Katello::Resources::Candlepin::UpstreamConsumer.get
```

Comment:
Candlepin APIs accept an 'includes' parameter in order to filter out the attributes returned in the response. Since we don't actually need any of the data here it'd be great to add that ability to .get so we can get only a single attribute for performance reasons.

You can refer to existing examples in the resource classes but the usage would end up something like this:

```
::Katello::Resources::Candlepin::UpstreamConsumer.get(includes: [:uuid])
```